index.html

```diff
@@ -2538,6 +2544,32 @@ <h5>Cover</h5>
     …
 }</pre>
 
+						<pre class="example" title="A purely decorative cover.">{
+    …
+    "resources" : [
+        {
+            "type"           : "LinkedResource",
+            "url"            : "cover.jpg",
+            "encodingFormat" : "image/jpeg",
+            "rel"            : "cover",
+            "name"           : "",
+        },
+        …
+    ],
+    …
+}</pre>
+
+						<p>If a user agent requires alternative text for a cover image to make an interface accessible,
+							and the <code>name</code> property is not specified, it MAY attempt to construct the
+							alternative text from the publication metadata. This specification does not mandate how such
+							alternative text is created. One method is to construct the alternative text as a string
+							that identifies that the image as the cover, followed by the <a href="#pub-title"
+								>publication title</a>.</p>
+
+						<p>Only one resource MAY be identified as the cover, but additional covers MAY specified using
+							the <a><code>alternate</code></a> property (e.g., to provide alternative dimensions or
+							resolution).</p>
+
```

Comment:
This sentence repeats the one right before example 37. Only one of them should remain; my guess is that is a better place for it, and the one above should be removed.

Reply:
Weird. I thought I corrected that in the merge conflict.